examples/Makefile

```diff
@@ -30,6 +30,9 @@ player-openal-buffer: player-openal-buffer.o
 player-sdl: player-sdl.o
 	$(LD) -o $@ $(LDFLAGS) $+ -lSDL $(LIBS)
 
+player-sdl2: player-sdl2.o
```

Comment:
The only difference player-sdl2.c over player-sdl.c is how SDL.h is included:
IMO, this one is not necessary: instead, we should change existing player-sdl.c
to use the SDL2 headers if required.

Besides, addition of this is out of the scope of this PR, no?

Reply:
I added the SDL2 version as I use it, and the SDL1 version isn't fine here. But you right, I'll strip this away and will post as another PR that turns the existing `player-sdl` into the thing that supports both SDL1 and SDL2, however, that will require one of the ways:
- bypass the `-I/path/to/include/SDL` or `-I/path/to/include/SDL2` depending on the desired build and -lsdl and -lsdl2. This is tricker as requires the knowledge of where headers actually stored, that's impossible without pkg-config or CMake
- use macros to specify which SDL header to use, but Makefile have both player-sdl and player-sdl2 targets, this is a simpler solution that doesn't require major changes, and will work for both

Reply:
You can use `pkg-config --libs|--cflags sdl[2]` or the conventional `sdl[2]-config --libs|--cflags`

Reply:
And yes, it should be a subject of another patchset

Reply:
Yeah, the pkg-config way seems fine, I'll try to apply, but the last question: do you want I'll post that as separated PR, or it's fine it will appear here?

Reply:
Please, _not_ here.

Reply:
Got it, will do. These commits I'll drop from this branch.